.eslintrc.js

```diff
@@ -21,33 +23,52 @@ module.exports = {
     '**/ios/main.jsbundle',
     '**/stylelint.config.js',
     '**/dist/',
-    '**/lib-dist/'
+    '**/lib-dist/',
+    '.eslintrc.js'
   ],
   rules: {
-    'arrow-body-style': 'error',
-    'arrow-parens': ['error', 'as-needed'],
+    // Overly strict rules (for now)
+    'class-methods-use-this': 'off',
+    'default-case': 'off',
+    'global-require': 'off',
+    'import/extensions': 'off',
+    'import/no-extraneous-dependencies': 'off',
+    'import/no-unresolved': 'off',
+    'import/prefer-default-export': 'off',
+    'lines-between-class-members': 'off',
+    'no-shadow': 'off',
+    'no-nested-ternary': 'off',
```

Comment:
Noooooo :sob: 

Seriously, I would think this could really make sense to have as a rule.

Reply:
Added to TODOs, will do in a separate PR. If you want you can do it as well :)

Reply:
Done in #447